README.md

```diff
@@ -1,94 +1,46 @@
-5. Create a branch with the name as specified in the [Naming Conventions: branch name](#branch-name).
-6. Fix the issue in the new branch.Remember to adhere to coding style and testing requirements of the project, 
-   as specified in the project documentation.
-7. When you think you are done with fix, 
-   1. Push the code to your fork.
-   2. Create a PR. 
-   
-     > You may create a PR even before you are done with the fix, if you want to seek some early feedback from the dev team.
-8. After creating the PR, post a comment to inform the dev team that it is ready for review. <br>
-   If you do not get any response from the dev team within 1-2 days, keep posting reminders in the PR thread.
-   
-## Submitting subsequent PRs
+* [**Overseeing Issue/PR Life Cycle**](docs/OverseeingIssuePrLifeCycle.md) : Instructions for team/area leads 
+  on overseeing issue/PR life cycles.
+* [**Managing Releases**](docs/ManagingReleases.md) : Instructions for managing releases, milestones, 
+  and project schedule.
+* [**Defining Labels**](docs/DefiningLabels.md) : Instructions for merging a PR.
 
-* After you have managed to successfully merge one PR, you can gradually move to harder issues. 
-* As harder issues take longer to finish, it is prudent to post a message in the issue to let others know that you are 
-  working on an issue.
-* Try to adopt [Git Usage Guidelines](#git-usage-guidelines).
+<br>
 
-# Coding Standards
+**Contributing to this repo**: We welcome contributions. Just follow the instructions in the above links to submit 
+your contributions as a PR.
 
-* [Java](https://oss-generic.github.io/process/codingstandards/coding-standards-java.html)
-* [HTML](https://oss-generic.github.io/process/codingstandards/coding-standards-html.html)
+**License** : MIT
 
-# Naming Conventions
-
-##### Branch name
-
-format: **`issueNumber-some-keywords-from-issue-title`**
-
-e.g. if the issue title is `Error alert email has very long subject #5958`, <br>
-your branch name can be `5958-error-alert-long-subject`
-
-##### PR title
-
-format: **`IssueTitle #IssueNumber`**
-
-e.g. `Error alert email has very long subject #5958`
-
-# Git Usage Guidelines
-
-* Commit each distinct change as a separate commit. You may have to `squash` trivial commits.
-* Write short descriptive commit message 
-  
-  > It's common practice to use the imperative mood when writing commit messages <br>
-  e.g. `Add README.md` rather than `Added README.md` or `Adding README.md`
-* Rebase your branch if the master branch has progressed since you started your branch. <br>
-  Alternatively (less preferred), you can merge the master branch to your branch.
-
-**Resources**
-
-1. [Web article] [How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/)
-
-# Contributing to this Repo
-
-We welcome contributions. Just follow the process described above to submit your contributions as a PR.
\ No newline at end of file
+[gh]: https://github.com/
```

Comment:
I don't think this is necessary; there are no anchors to this anywhere on the file.